pages.de/common/gobuster.md

```diff
@@ -0,0 +1,28 @@
+# gobuster
+
+> Findet versteckte Pfade auf Webservern und mehr.
+> Weitere Informationen: <https://github.com/OJ/gobuster>.
+
+- Finde Verzeichnisse und Dateien, die den Wörtern der Wortliste entsprechen.
+
+`gobuster dir --url {{https://example.com/}} --wordlist {{pfad/zu/datei}}`
```

Comment:
```suggestion
`gobuster dir --url {{https://beispiel.com/}} --wordlist {{pfad/zu/datei}}`
```
Same for the other examples

Reply:
yea, sure, i guess, we can do that too, tysm